runtime/jcl/common/mgmtthread.c

```diff
@@ -1317,15 +1327,39 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 	j9object_t monitorOwnerObject = NULL;
 	IDATA exc = 0; /* exception index */
 
+#if JAVA_SPEC_VERSION >= 19
+	J9VMThread stackThread = {0};
+	J9VMEntryLocalStorage els = {0};
+	J9VMContinuation *continuation = NULL;
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
the thread object can be stored in a local to reduce the number of times it is fetched from memory.
```suggestion
	j9object_t threadObject = currentThread->carrierThreadObject;
#else /* JAVA_SPEC_VERSION >= 19 */
	j9object_t threadObject = currentThread->threadObject;
#endif /* JAVA_SPEC_VERSION >= 19 */
```